packages/extension-api/src/extension-api.d.ts

```diff
@@ -1235,10 +1252,10 @@ declare module '@podman-desktop/api' {
     ): Promise<R>;
 
     /**
-     * Show OS desktop notification
-     * @param options
+     * Show notification on different area of Podman Desktop based on its options (Dashboard, bell icon list, OS notification)
+     * @param options define how the notification must be created.
      */
-    export function showNotification(options: NotificationOptions): Disposable;
+    export function showNotification(options: NotificationCardOptions | NotificationOptions): Disposable;
```

Comment:
hello, I'm wondering if we need another interface or we just enhance NotificationOptions ?

for example missing type can default to 'info' type

Reply:
and only if it's added with higlight that we display it in dashboard as well

Reply:
the problem is that notificationOptions has all properties optionals, so by keeping them separated you have a check while coding. I expect that if a notification has to be shown in the bell list or in the dashboard it has a title and a description. But to keep it compatible with the old NotificationOptions i have to keep all props optional and it would be ugly to return an error if we find that the highlight is set but it has no title or description, or to not show it in the bell list bc it misses some props, no?

Reply:
if there is no title it can add the extension's name

so we never throw any error, we just display it with defaults. If it looks odd it's the extension's responsibility.

Reply:
I'm +0 on the change. I would say let's keep it simple with only extra attributes but if other reviewers/yourself think it's better to use two interfaces I'm ok.

Reply:
removed the second interface